apis/k8ssandra/v1alpha1/k8ssandracluster_types.go

```diff
@@ -457,26 +461,84 @@ type EmbeddedObjectMeta struct {
 	Name string `json:"name"`
 
 	// +optional
-	Labels map[string]string `json:"labels,omitempty"`
+	Metadata *CassandraDatacenterMeta `json:"dcMetadata,omitempty"`
+}
 
+type CassandraDatacenterMeta struct {
```

Comment:
I would move this type to the `meta` package as well. But wouldn't fight for it :-)